docs/content/docs/deployment/azure-frs.md

```diff
@@ -1,5 +1,94 @@
+
+Let's take a look at how to go about using the `frs-client` in your app!
+
+# Connecting to the Service
+
+To connect to our FRS instance, we first need to instaniate our `FrsClient`. This takes in, as configuration parameters, the tenant ID, orderer, and storage URLs that were provided as part of the FRS onboarding process. It also requires a token provider to generate the JWT token that will be used to authorize the current user against the service. The `InsecureTokenProvider` should only be used for testing purposes as it exposes the tenant key secret in your client-side code bundle. This should be replaced with an implementation of `ITokenProvider` that fetches the token from your own backend service that is responsible for signing it with the tenant key. 
+
+```typescript
+const config: FrsConnectionConfig = {
+    tenantId: "myFrsTenantId",
+    // IMPORTANT: this token provider is suitable for testing ONLY. It is NOT secure.
+    tokenProvider: new InsecureTokenProvider("myFrsTenantKey", { id: "UserId", name: "Test User" }),
+    orderer: "https://myFrsOrdererUrl",
+    storage: "https://myFrsStorageUrl",
+}
+
+const client = new FrsClient(config);
+```
+
+Now that we have an instance of our client, we can start using it to create new or load existing Fluid containers!
+
+# Managing Containers
+
+The `FrsClient` provides two functions on its API to create and get containers respectively. They both take in two parameters:
+- A container config that defines the ID of the container and an optional entrypoint for logging
+- A container schema that defines the DDSes and data objects that this container will hold
+
+```typescript
+const schema = {
+    name: "my-container",
+    initialObjects: {
+        /* ... */
+    },
+    dynamicObjectTypes: [ /*...*/ ],
+}
+const frsClient = new FrsClient(config);
+await frsClient.createContainer({ id: "_unique-id_" }, schema);
+const { fluidContainer, containerServices } = await frsClient.getContainer({ id: "_unique-id_" }, schema);
+```
+
+The ID that is being passed in to the container config can be thought of as the "filename" for the item holding the data for this container on the FRS backend. Any client that wants to join the same collaborative session just needs to call `getContainer` with the same container ID.
```

Comment:
Let's not introduce a file type metaphor since it's really only useful if the developer is coming in with files as their default paradigm.

```suggestion
The `id` being passed into the container config is a unique identifier to a container instance. Any client that wants to join the same collaborative session just needs to call `getContainer` with the same container `id`.
```